hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/PrefixManagerImpl.java

```diff
@@ -260,9 +254,6 @@ public OMPrefixAclOpResult addAcl(OzoneObj ozoneObj, OzoneAcl ozoneAcl,
     // update the in-memory prefix tree
     prefixTree.insert(ozoneObj.getPath(), prefixInfo);
 
-    if (!isRatisEnabled) {
```

Comment:
I'm not sure about this one. This code path (isRatisEnabled=false) is used by OmSnapshotManager. Are we sure it doesn't break the Snapshot feature?

Reply:
Restored.

Reply:
@hemantk-12 or @smengcl do you know why Snapshot creates PrefixManagerImpl assuming ratis not enabled?